readme.md

```diff
@@ -27,10 +33,19 @@ github "sindresorhus/LaunchAtLogin"
 
 Add a new ["Run Script Phase"](http://stackoverflow.com/a/39633955/64949) below "Embed Frameworks" in "Build Phases" with the following:
 
+SwiftPM:
+
+```
+${BUILD_ROOT}/../../SourcePackages/checkouts/LaunchAtLogin/LaunchAtLogin/copy-helper.sh
```

Comment:
I would try to find an environment variable where you don't have to use `../..`. That's very fragile to future changes.

https://gist.github.com/gdavis/6670468

Reply:
I looked for one but I find an updated version of all environment variables.

Reply:
You can just print the environment variables yourself. https://www.google.com/search?q=xcode+print+environment+variables&oq=xcode+print+environ&aqs=chrome.0.0j69i57.3568j0j0&sourceid=chrome&ie=UTF-8

Reply:
This path is also not correct. The `BUILD_ROOT` during archiving is not there. It needs 5 levels of `../`. And even then, the script itself needs to be updated to support the SwiftPM locations.